lab_3/main.py

```diff
@@ -59,7 +87,15 @@ def encode_corpus(storage: LetterStorage, corpus: tuple) -> tuple:
     :param corpus: a tuple of sentences
     :return: a tuple of the encoded sentences
     """
-    pass
+    if not isinstance (corpus,tuple) or not  isinstance(storage,LetterStorage):
+        return ()
+    encoded_corpus=[]
+    for sent in corpus:
+        encoded_sent=[]
+        for letter in sent:
```

Comment:
a corpus consists of sentences
sentences consist of words
words consist of letters